controllers/provider-packet/example/30-etcd-backup-secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
Packet doesn't have support for backups yet, so I wouldn't try to fake it here. Remove this file.

Reply:
done